dist/images/ovnkube.sh

```diff
@@ -764,6 +764,11 @@ ovn-master() {
   }
 
   echo "=============== ovn-master ========== MASTER ONLY"
+  ovn_metrics_bind_address="0.0.0.0"
+  if [[ "${net_cidr}" == *":"* ]]; then
+    ovn_metrics_bind_address="[::]"
```

Comment:
This shouldn't matter; golang treats `0.0.0.0` and `::` identically for `Listen`

Reply:
OK, I'll undo this change.

Reply:
Done.